tests/integration/module.test.js

```diff
@@ -0,0 +1,166 @@
+import { beforeAll, describe, expect, test } from '@jest/globals'
+
+import { HOST, PORT, ELECTRUM_PORT, ZMQ_PORT, DATA_DIR } from '../config.js'
+
+import { BitcoinCli, Waiter } from '../helpers/index.js'
+
+import WalletManagerBtc from '../../index.js'
+
+const abs = x => x < 0n ? -x : x
+
+function parseRawTransaction (rawTransaction, recipientAddress) {
+  const getAddress = (vout) => vout.scriptPubKey.address || vout.scriptPubKey.addresses?.[0]
+  
+  const output = rawTransaction.vout.find(vout => getAddress(vout) === recipientAddress)
+  
+  return {
+    txid: rawTransaction.txid,
+    details: [{ address: getAddress(output), amount: output.value, vout: 0 }]
+  }
+}
+
+const fees = {
+  44: 226n,
+  84: 141n
+}
+
+const SEED_PHRASE = 'cook voyage document eight skate token alien guide drink uncle term abuse'
+
+const bitcoin = new BitcoinCli({
+  host: HOST,
+  port: PORT,
+  zmqPort: ZMQ_PORT,
+  dataDir: DATA_DIR,
+  wallet: 'testwallet'
+})
+
+const waiter = new Waiter(bitcoin, {
+  host: HOST,
+  electrumPort: ELECTRUM_PORT,
+  zmqPort: ZMQ_PORT
+})
+
+describe.each([44, 84])('@wdk/wallet-btc (BIP %i)', (bip) => {
+  const CONFIGURATION = {
+    host: HOST,
+    port: ELECTRUM_PORT,
+    network: 'regtest',
+    bip
+  }
+
+  let wallet, account0, account1
```

Comment:
Remove global state variables account0 and account1:
```suggestion
  let wallet
```